bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -31,7 +31,7 @@ metadata:
     categories: Logging & Tracing
     certified: "false"
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2023-04-10T17:45:14Z"
+    createdAt: "2023-04-28T17:51:30Z"
```

Comment:
If think, this file needs to be reverted.

Reply:
Done.